www/static/js/index.js

```diff
@@ -145,17 +145,48 @@ $(document).ready(function () {
 
     // Smooth scroll to anchor links
     $("a[href^='#']").on('click', function(e) {
-        if(this.hash) {
+
+        // scroll only if there is a hash in the href
+        var hash = this.hash;
+        if (hash) {
 
             // prevent default anchor click behavior
             e.preventDefault();
 
-            // store hash
-            var hash = this.hash;
+            // get the fragment without the "#" symbol
+            var targetName = hash;
+            if (targetName[0] === "#") {
```

Comment:
I guess the condition will never fail as long as browsers obey the standard. I'll remove the check.